.github/workflows/go.yml

```diff
@@ -22,59 +26,58 @@ jobs:
     name: "check code format"
     runs-on: ubuntu-latest
     steps:
```

Comment:
vscode auto-formatted the yaml a bit when I touched this. I think this is more consistent with the format in `ci.yaml` and a bit more human-readable, so I'm inclined to keep it. Let me know if you'd rather not, and I can take the whitespace adjustments out.